src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -498,8 +502,8 @@ def find_fixed_range(uworker_input):
   if result and not result.is_crash():  # pylint: disable=no-member
     error_message = (
         f'Known crash revision {known_crash_revision} did not crash')
-    progression_task_output.crash_revision = int(max_revision)
-    return uworker_io.UworkerOutput(
+    progression_task_output.crash_revision = int(min_revision)
```

Comment:
Should be `max_revision`?

Reply:
I'd need @jonathanmetzman or @oliverchang to weigh in here. Maybe it's intentional, because in this branch we found that the target did *not* crash at `min_revision`?